gke/README.md

```diff
@@ -51,8 +51,34 @@ The following describes the setup process for this project:
 1. Install Jenkins on GKE using the stable/jenkins helm chart: [Jenkins On GKE Tutorial](
    https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial).
 
-1. Install and configure the GKE plugin: [Jenkins GKE Docs](
-   https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+1. Install and configure the following plugins:
+   1. GKE plugin: [Jenkins GKE Docs](https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+   1. GCS plugin: [Jenkins GCS Docs](https://github.com/jenkinsci/google-storage-plugin/blob/develop/README.md).
```

Comment:
Could you please also add the OAuth plugin to this list: https://github.com/jenkinsci/google-oauth-plugin/blob/develop/README.md

Reply:
done.